src/__tests__/lib/exampleData.js

```diff
@@ -114,6 +114,8 @@ export const realm = 'https://zulip.example.org';
 
 export const zulipVersion = '2.1.0-234-g7c3acf4';
 
+export const zulipFeatureLevel = 100;
```

Comment:
Nit in commit message for

e3ca22d27 RealmScreen: Include feature level from /server_settings in REALM_ADD.

"accessed"